scripts/clickhouse_operator_sync.sh

```diff
@@ -62,16 +62,16 @@ kubectl-slice -f "$TMP_FOLDER/clickhouse-operator.yaml" -o "${CHART_PATH}/templa
 FILES="${CHART_PATH}/templates/clickhouse-operator/*"
 for f in $FILES
 do
-    sed -i'' '1i\
```

Comment:
This is a fix for a regression that was recently introduced via #371 (the regression is fixed on macOS) cc @hazzadous

Reply:
This breaks gnu sed support I believe

Reply:
TIL https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux